app/controllers/v1/calls_controller.rb

```diff
@@ -0,0 +1,10 @@
+class V1::CallsController < V1::BaseController
+
+  def create
+    person = Person.find_or_create_by(email: params[:email], phone: params[:phone])
```

Comment:
this will fail if email is already in system with different phone. 
Should probably be `Person.create_with(phone: params[:phone])).find_or_create_by(email: params[:email])`